src/main.ts

```diff
@@ -9,4 +9,4 @@ if (environment.production) {
 }
 
 platformBrowserDynamic().bootstrapModule(AppModule)
-  .catch(err => console.error(err));
+.catch(err => console.error(err));
```

Comment:
ten plik nie ma dla nas żadnego znaczenia, bo nigdy nie będzie edytowany, ale podobało mi się to wcięcie ;c

Reply:
Jest na to opcja, już popatrzę - nie wiem, czy wtedy się nie wcinały przypadkiem jakieś inne rzeczy bez sensu